CHANGES.md

```diff
@@ -2760,7 +2761,7 @@
 ### 2.3.0
 
 - Share links now contain details about the picked point, picked features and currently selected feature.
-- Reorganised the display of disclaimers so that they're triggered by `CatalogGroup` and `CatalogItem` models, which trigger `terria.disclaimerEvent`, which is listened to by DisclaimerViewModel`. `DisclaimerViewModel` must be added by the map that's using Terria.
+- Reorganised the display of disclaimers so that they're triggered by `CatalogGroup` and `CatalogItem` models, which trigger `terria.disclaimerEvent`, which is listened to by DisclaimerViewModel`.`DisclaimerViewModel` must be added by the map that's using Terria.
```

Comment:
```suggestion
- Reorganised the display of disclaimers so that they're triggered by `CatalogGroup` and `CatalogItem` models, which trigger `terria.disclaimerEvent`, which is listened to by DisclaimerViewModel`. `DisclaimerViewModel` must be added by the map that's using Terria.
```

Re-add erroneously deleted space.